src/settings/urlSettings.ts

```diff
@@ -64,6 +64,7 @@ export const ACCOUNT_PROFILE_QUESTIONS_URL = `${ACCOUNT_ROOT_URL}/questions/:wor
 export const ADMIN_IA_WORLD_BASE_URL = `${ADMIN_ROOT_URL}/w`; // e.g. /admin/w
 export const ADMIN_IA_WORLD_CREATE_URL = `${ADMIN_ROOT_URL}/create-world`; // e.g. /admin/create-world
 export const ADMIN_IA_WORLD_PARAM_URL = `${ADMIN_IA_WORLD_BASE_URL}/:worldSlug`; // e.g. /admin/w/world123
+export const ADMIN_IA_WORLD_TOOLS = `${ADMIN_IA_WORLD_PARAM_URL}/tools`; // e.g. /admin/w/world123/tools
```

Comment:
The naming convention is described in the file itself, lines 29 to 31

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2747/commits/73bc0106ad8f80a3da7b69b082166b60d78fc4e8